Makefile

```diff
@@ -6,12 +6,14 @@ install: build force_link
 
 build: $(OUT_DIR)/cry
 
-$(OUT_DIR)/cry: src/cry.cr
+$(OUT_DIR)/cry: src/** lib
```

Comment:
Why is this necessary?

Reply:
After the : come the dependencies.
The compiled binary 'cry' depends on its sources and the lib/ directory ("depends" meaning it needs to be recompiled if the resources it depends on change).
The src/** is needed because without it, if you changed src/cry/command.cr, 'make' wouldn't realized cry had to be recompiled (and would tell you "Nothing to be done for 'all'" when you called it).
lib/ is needed because the compilation cannot begin before lib/ being present. (And conveniently, if lib/ isn't present then make looks up the "lib" target and runs "shards" to create it, estabilshing the proper dependency chain going from the final deliverable ("cry" binary) to the components necessary for building.)